pkgs/applications/window-managers/dwm/default.nix

```diff
@@ -18,6 +20,10 @@ stdenv.mkDerivation {
   # Allow users set their own list of patches
   inherit patches;
 
+  # Allow users to set the config.def.h file containing the configuration
+  postPatch = let configFile = if isDerivation conf || builtins.isPath conf then conf else writeText "config.def.h" conf;
+  in optionalString (conf!=null) "cp ${configFile} config.def.h";
+
   buildPhase = " make ";
```

Comment:
I just wonder is buildPhase really needed here?

Reply:
I'm not sure, if not should it be removed in this same PR? (I'm quite new to this)

Reply:
Just tested it, removing it still builds dwm just fine.